### PR TITLE
Adding Hoodie

### PR DESCRIPTION
**Pull request by gr2m**
I hope I did this right :) We love new contributors and put a lot of efforts to maintain a list of starter issues at all times.

Question: Is it possible to set two labels? `starter` is the label we add to issues that are perfect for new contributors. `ready` is an additional label which means it's up for grab. Once we remove `ready` it’s assigned to someone.

### CODE REVIEW

File: _data/projects/Hoodie.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Hoodie
+desc: Generic Backend for frontend Apps
+site: https://github.com/dahlbyk/up-for-grabs.net
```
Comment (shiftkey) [inline _data/projects/Hoodie.yml]: Set this to your project's home page :grin:

File: _data/projects/Hoodie.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+tags:
+- javascript
+- web
+- offlinefirst
+- nobackend
+upforgrabs:
+  name: starter
+  link: http://go.hood.ie/hoodie-starter-issues
```
Comment (shiftkey) [inline _data/projects/Hoodie.yml:11]: We've got discussions about tracking multiple repositories #244 and multiple labels #240 but this is a fine starting point

**Comment by shiftkey** [comment]
@gr2m welcome! I added a couple of notes here - it's pretty close to merging!

> Is it possible to set two labels? `starter` is the label we add to issues that are perfect for new contributors. `ready` is an additional label which means it's up for grab. Once we remove `ready` it’s assigned to someone.

I linked to a couple of related issues above, but we haven't quite got to the point of proposing changes to the markup structure (and then putting this onto the site). If you've got some ideas for tackling these I'd love to hear from you in those discussions...

**Comment by gr2m** [comment]
How about you would support this format for multiple tags:

```
name: starter, ready
```

With the comma-separated list you would not break the current format. 

Is the PR okay like this now?

**Comment by shiftkey** [comment]
@gr2m currently on the site we use the label here, just below the project name:



The original intent here is for projects to have a label that represents issues which they think are good for new contributors - so even though the site is called Up For Grabs, we don't dictate that you use that label.

And the other big benefit here is that, if you're using a GitHub link to represent your issues we can call the API and get the issue count from the JSON response. By adding in more flexibility in the format, we lose the ability to work with raw data and explore more project statistics - which would be cool to do on the site itself.

Anyway, we haven't settled on anything just yet - I have a side-project exploring this over at https://github.com/shiftkey/up-for-grabs-api-demo/ - and supporting different project workflows is something I'll keep in mind...

**Comment by shiftkey** [comment]
> Is the PR okay like this now?

LGTM. Thanks!

File: _data/projects/Hoodie.yml (hunk 2 above)
Comment (gr2m) [inline _data/projects/Hoodie.yml:11]: How about this URL: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+org%3Ahoodiehq+label%3Astarter+label%3Aready+ – does that work with your labels showing the number of open issues?

Comment (shiftkey) [inline _data/projects/Hoodie.yml:11]: It won't display the issue count because of the `org:hoodiehq` parameter

**Pull request merged**